terraform/container-app/functions.tf

```diff
@@ -70,20 +70,9 @@ resource "azurerm_linux_function_app" "contentful_function" {
       app_settings["AzureWebJobsStorage__clientId"],
       app_settings["AzureWebJobsStorage__credential"],
       app_settings["WEBSITE_RUN_FROM_PACKAGE_BLOB_MI_RESOURCE_ID"],
-
+      app_settings,
```

Comment:
If the whole appsettings is here do the ones above it need to be?

Reply:
Absolutely not... But I left them there because I was scared. I'll remove them now though, good spot